docs/dev/migrating-to-v1.rst

```diff
@@ -362,19 +367,20 @@ has not been changed compared to the GridModel.
 +------------------------------+-------------------------------------------+
 | v0.x                         | v1                                        |
 +==============================+===========================================+
-| model.set_grid(...)          | model.grid.set(...)             |
+| model.set_grid(...)          | model.grid.set(...)             		   |
 +------------------------------+-------------------------------------------+
-| model.read_grid(...)         | model.grid.read(...)            |
+| model.read_grid(...)         | model.grid.read(...)            		   |
 +------------------------------+-------------------------------------------+
-| model.write_grid(...)        | model.grid.write(...)           |
+| model.write_grid(...)        | model.grid.write(...)            		   |
 +------------------------------+-------------------------------------------+
-| model.setup_grid(...)        | model.grid.create(...)          |
+| model.setup_grid(...)        | model.grid.create(...)          		   |
 +------------------------------+-------------------------------------------+
-| model.setup_grid_from_*(...) | model.grid.add_data_from_*(...) |
+| model.setup_grid_from_*(...) | model.grid.add_data_from_*(...) 		   |
 +------------------------------+-------------------------------------------+
```

Comment:
Alignment still looks off? Or is it just this view?

Reply:
It is just the view, if you check this file in the branch the table looks fine